src/js/components/search/filters/defc/DEFCheckboxTreeLabel.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const DEFCheckboxTreeLabel = ({
+    label,
+    subLabel,
+    value
+}) => {
+    if (label.includes('|')) {
+        const labels = label.split('|');
+        const subLabels = subLabel.split('|');
+        return (
+            <div className="checkbox-tree-label">
+                <div className="checkbox-tree-label__value-container">
+                    <div className="checkbox-tree-label__value-container-value">
+                        {value}
+                    </div>
+                </div>
+                {labels.map((_, i) => (
```

Comment:
Any reason we are going back up the lexical scope to reference `labels[i]` when `_` is the same thing?

<img width="584" alt="Screen Shot 2020-07-29 at 4 37 23 PM" src="https://user-images.githubusercontent.com/14301977/88850916-05d21e00-d1ba-11ea-82ee-b6e2f91330de.png">